bin/create-pr-from-issue.rb

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env ruby
+
+require_relative "../lib/github_collaborators"
+
+# Grab GitHub token
+token = ENV.fetch("ADMIN_GITHUB_TOKEN")
+
+# GitHub settings
+owner = "ministryofjustice"
+repository = "github-collaborators"
+
+# Returns block needed to insert into a collaborator file
+# Pass it the infomation needed for such, see example at:
+# https://github.com/ministryofjustice/github-collaborators/blob/main/terraform/yjaf-auth.tf
+def return_insert(gh_user, perm, name, email, org, reason, added_by, review_after)
+    <<~EOF
+    \s\s\s\s{
+        \s\sgithub_user  = "#{gh_user}"
+        \s\spermission   = "#{perm}"
+        \s\sname         = "#{name}"         
+        \s\semail        = "#{email}"        
+        \s\sorg          = "#{org}"          
+        \s\sreason       = "#{reason}"       
+        \s\sadded_by     = "#{added_by}"     
+        \s\sreview_after = "#{review_after}" 
+    \s\s\s\s},
+    EOF
+end
+
+# Creates a branch and commits all changed files
+# This probably deserves its own class but keeping it here for now until we need more major functionality in this area
+def create_branch_for_pr
+    # Init local Git
+    g = Git.open(".")
+  
+    g.config("user.name", "Operations Engineering Bot")
+    g.config("user.email", "dummy@email.com")
```

Comment:
Do we want this to be our team email?

Reply:
Did think about that - kept it as this at the moment because it doesn't seem to respect that actual config - e.g. it uses its default ones still, it's something I need to look into.